### ISSUE TITLE
exporting `transform`

### ISSUE THREAD

**Issue by gdkrmr** [JuliaStats/StatsBase.jl#453]
#452 is made worse by the fact that `StatsBase` does not export `transform` but `MultivariateStats` does and therefore function overloading does not work.

**Comment by nalimilan** [comment]
Can't MultivariateStats overload `StatsBase.transform` and export it?

**Comment by gdkrmr** [comment]
I guess that would be the solution

**Comment by rofinn** [comment]
Yeah, I feel like `transform` is a general enough term that we shouldn't export it. `using StatsBase: transform` is easy enough to do.

**Comment by nalimilan** [comment]
See also https://github.com/JuliaData/TableOperations.jl. Cc @quinnj.
